src/zoid/buttons/util.js

```diff
@@ -357,3 +360,60 @@ export function getButtonSize(
     }
   }
 }
+
+export const getModal: (
+  clientID: string,
+  merchantID: $ReadOnlyArray<string> | void
+) => Object = memoize(async (clientID, merchantID) => {
+  try {
+    const namespace = getNamespace();
+    if (!window[namespace].MessagesModal) {
+      const modalBundleUrl = () => {
```

Comment:
can we move this up to a top-level function declaration, called something like `buildModalBundleUrl`